### DISCUSSION TITLE
Physical mouse is not responding

### DISCUSSION THREAD

**Discussion by shiumano** [ppy/osu#24808] [Q&A]
### Type

Other

### Bug description

The physical mouse is not responding; it does not capture even when raw input is turned on.

### Screenshots or videos

https://github.com/ppy/osu/assets/75152752/8d8981a6-54a4-4384-b505-f0e7bf071dfc

### Version

2023.913.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/12597705/database.log)
[input.log](https://github.com/ppy/osu/files/12597706/input.log)
[network.log](https://github.com/ppy/osu/files/12597707/network.log)
[performance.log](https://github.com/ppy/osu/files/12597708/performance.log)
[runtime.log](https://github.com/ppy/osu/files/12597710/runtime.log)

**Comment by peppy** [comment]
Android
